reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md

```diff
@@ -253,7 +253,7 @@ the collection.
 ```
 
 Starting in PowerShell 3.0, if you request the **Count** or **Length** property
-of zero objects or one object, PowerShell returns the correct value.
+of singleton objects that aren't collections.
```

Comment:
It looks like this got turned into a sentence fragment.

Reply:
fixed